### PR TITLE
Dependency updates + some improvements

### PR DESCRIPTION
**Pull request by smoench**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
     "type": "library",
     "license": "MIT",
     "require": {
```
Comment (DavidBadura) [inline composer.json]: can you add the minimal php version too?

Comment (smoench) [inline composer.json]: done

**Comment by timglabisch** [comment]
ping @meandmymonkey can you give me access to this repo :) ?

**Comment by timglabisch** [comment]
👍 looks good, feel free to merge =)

File: composer.json [hunk 2]
```diff
@@ -13,7 +14,7 @@
         }
     ],
     "require-dev": {
-        "phpunit/phpunit": "4.8.*"
+        "phpunit/phpunit": "^5.5"
```
[the remark this answers is not in the document]
Comment (smoench) [reply composer.json]: Should be another PR with migrating to namespaced TestCase

**Pull request merged**